100-level-AsyncAPI-Intro/step2.md

```diff
@@ -0,0 +1,124 @@
+## Request/reply 
+As Eve and Chan had almost finished setting up the communication system, Eve explained the request/reply pattern to Chan, which is not a component but a typical communication pattern. 
+
+Eve explains to Chan that this is a type of communication pattern where there are two actors, a requestor and a replier. 
+
+Here, they set up a channel called `communication`, which is used to exchange messages, send requests asking how Eve's doing, what she's up to, and general chit-chat or cool discoveries by Chan as he travels around the galaxy. The reply address where the response to his request needs to be delivered is known while defining the communication system.
```

Comment:
```suggestion
As Eve and Chan near the completion of their communication system setup, Eve takes a moment to elaborate on the request/reply pattern to Chan, emphasizing that it is a communication pattern rather than a separate component.

Eve clarifies to Chan that this pattern involves two key roles: a requestor and a replier.

In their setup, they establish a channel named `communication` to facilitate requests and enable communication. Now they can stay in touch and communicate about Eve's adventures and Chan's fascinating discoveries during his intergalactic travels. 
```